src/kili/services/plugins/tools.py

```diff
@@ -0,0 +1,17 @@
+"""
+Set of common functions used by plugins
+"""
+
+DUPLICATE_ERROR_SUBSTRING = "an entity Plugin already exists with value"
+
+
+def check_errors_plugin_upload(result, file_path, plugin_name):
+    """
+    Check if the error during upload is because the plugin already
+    exists and print the upload function for the user
+    """
+    if "errors" in result and DUPLICATE_ERROR_SUBSTRING in result["errors"][0]["message"]:
+        print(
+            f'Perhaps you wanted to use update_plugin("{file_path}", "{plugin_name}") '
```

Comment:
```suggestion
            f'Perhaps you wanted to use kili.update_plugin(file_path="{file_path}", plugin_name="{plugin_name}") '
```
And perhaps handle the case where one of those variables is not defined ?

Reply:
Only plugin_name can be undefined but the method that calls this function set it as the file name so no need to check it